### DISCUSSION TITLE
Price VAT Information

### DISCUSSION THREAD

**Discussion by cdigruttola** [shopperlabs/framework#238] [Ideas]
Good morning,
I'm searching a way to set the VAT for the product, but I cannot find it.
Is there a possibility to add this feature?
Thanks

**Comment by mckenziearts** [comment]
Hey @cdigruttola, sorry for the late response on this.

Tax handling has landed since this was posted: you can now configure tax zones, tax rates, and assign them to products or categories. See the Taxes docs at https://docs.laravelshopper.dev/v2/user-guide/settings/taxes for the setup flow.

If something is still missing for your specific use case, open a new discussion with the scenario and we'll take a look.
